includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * WooCommerce Admin Setup Wizard Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class adds actions to track usage of the WooCommerce Onboarding Wizard.
+ */
+class WC_Admin_Setup_Wizard_Tracking {
+	/**
+	 * Init tracking.
+	 */
+	public static function init() {
+		if ( empty( $_GET['page'] ) || 'wc-setup' !== $_GET['page'] ) { // WPCS: CSRF ok, input var ok.
+			return;
+		}
+
+		add_action( 'admin_init', array( __CLASS__, 'track_store_setup' ), 1 );
```

Comment:
How about calling a function here that has a switch statement with the logic of `is_saving_step`?